classes/class-kp-assets.php

```diff
@@ -0,0 +1,176 @@
+<?php
+/**
+ * Used for inserting JavaScript and CSS files, conditionally.
+ *
+ * @package WC_Klarna_Payments/Classes
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * KP_Assets class.
+ */
+class KP_Assets {
+
+	/**
+	 * Class constructor.
+	 */
+	public function __construct() {
+		/* Klarna Express Checkout (aka Express Button). */
+		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_express_button' ) );
+		add_action( 'script_loader_tag', array( $this, 'express_button_script_tag' ), 10, 2 );
+		add_action( 'woocommerce_proceed_to_checkout', array( $this, 'express_button_placement' ) );
+	}
+
+	/**
+	 * Conditionally enqueue the scripts requried for Express Button.
+	 *
+	 * @return void
+	 */
+	public function enqueue_express_button() {
```

Comment:
possibly split this function into two, one for the script and one for the style. Just to keep them separated to make maintenance easier.